.github/workflows/e2e_cloud.yml

```diff
@@ -0,0 +1,88 @@
+name: E2E Cloud
+
+concurrency:
+  group: e2e-cloud-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - staging
```

Comment:
e2e cloud test should be only run main branch and run nightly

Reply:
Fixed on https://github.com/infinyon/mqtt-connector/pull/23/commits/51b7bb563cc21c78937e3782c04ac99ef8aaf32f